src/core/Env.js

```diff
@@ -288,6 +288,13 @@ getJasmineRequireObj().Env = function(j$) {
       return spyRegistry.spyOn.apply(spyRegistry, arguments);
     };
 
+    var ensureIsFunction = function(fn, caller) {
+      var fnType = Object.prototype.toString.call(fn);
```

Comment:
Done, thanks. I think it'd be valuable to include the type of the (invalid) argument, so I extract the Object.prototype.toString call out to a helper, j$.getType_.